fabtests/common/shared.c

```diff
@@ -536,14 +536,18 @@ static int ft_alloc_ctx_array(struct ft_context **mr_array, char ***mr_bufs,
 	return 0;
 }
 
-static void ft_set_tx_rx_sizes(size_t *set_tx, size_t *set_rx)
+static void ft_set_tx_rx_sizes(size_t *set_tx, size_t *set_rx, size_t *set_rx_msg_size)
 {
 	*set_tx = opts.options & FT_OPT_SIZE ?
 		  opts.transfer_size : test_size[TEST_CNT - 1].size;
 	if (*set_tx > fi->ep_attr->max_msg_size)
 		*set_tx = fi->ep_attr->max_msg_size;
 	*set_rx = *set_tx + ft_rx_prefix_size();
 	*set_tx += ft_tx_prefix_size();
+	*set_rx_msg_size = *set_rx;
+	if (ep)
+		fi_getopt(&ep->fid, FI_OPT_ENDPOINT, FI_OPT_MAX_MSG_SIZE,
```

Comment:
If you don't care about the return of `fi_getopt`, you need to add a `(void)` here to avoid raising coverity scan alarm